REFERENCE.md

```diff
@@ -886,25 +886,28 @@ Default value: `$redis::params::minimum_version`
 
 ### `redis::administration`
 
-Allows various adminstrative settings for Redis
+Allows various administrative settings for Redis
 As documented in the FAQ and https://redis.io/topics/admin
 
+For disabling Transparent Huge Pages please use forge module:
```

Comment:
nitpick/suggestion.  Your change eliminates `disable_thp`, and also all `thp` strings.  How about you make this line "For disabling Transparent Huge Pages (THP), please" etc etc.

No pressure, feel free to ignore.

Reply:
I actually think you are totally right, so I have added THP to make sure it's easier to search for.